src/rules/no-multiple-animated-style-usages.ts

```diff
@@ -0,0 +1,98 @@
+import { ESLintUtils } from "@typescript-eslint/experimental-utils";
+import { Scope } from "@typescript-eslint/experimental-utils/dist/ts-eslint";
+import { TSESTree } from "@typescript-eslint/experimental-utils/dist/ts-estree";
+
+export type Options = [];
+export type MessageIds = "NoMultipleAnimatedStyleUsagesMessage";
+
+const createRule = ESLintUtils.RuleCreator((name) => {
+  return `https://github.com/wcandillon/eslint-plugin-reanimated/blob/master/docs/${name}.md`;
+});
+
+const NoMultipleAnimatedStyleUsagesMessage =
+  "{{name}} cannot be used multiple times. Use separate useAnimatedStyle() calls instead.";
+
+const getVariableInScope = (
```

Comment:
This was based off of `isVarInScope()` in `rules/js-function-in-worklet.ts`, I don't know if there's a more idiomatic way to look up an in-scope variable?